VocaDbModel/Domain/ReleaseEvents/ReleaseEventEditableFields.cs

```diff
@@ -38,7 +38,9 @@ public enum ReleaseEventEditableFields {
 
 		Venue			= 16384,
 
-		WebLinks		= 32768
+		VenueEntry		= 32768,

```

Comment:
Here too.

Reply:
I think it would be better to change the old field to VenueName and the new field just Venue, doing the migration I mentioned.